nano/node/wallet.cpp

```diff
@@ -808,12 +808,13 @@ nano::public_key nano::wallet::deterministic_insert (nano::transaction const & t
 		{
 			work_ensure (key, key);
 		}
+		auto supply (wallets.node.online_reps.online_stake ());
+		auto half_principal_weight ((supply / 1000) / 2);
```

Comment:
Now that https://github.com/nanocurrency/nano-node/pull/2182 is merged, this should probably use `principal_weight_factor`